### PR TITLE
Docs/crypto plugin

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Update and expand docs wrt crypto plug-in mechanism.

- [x] update `ledger-integration.md (398d7ea)
- [X] ~update demo guides that require crypto~ (see #2301)

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: scripts/check_doc_links.py [hunk 1]
```diff
@@ -43,6 +43,15 @@
...
 
+IGNORE = {
+    "https://pypi.org/project/aea-crypto-fetchai/",
+    "https://pypi.org/project/aea-crypto-ethereum/",
+    "https://pypi.org/project/aea-crypto-cosmos/",
+    "https://github.com/fetchai/agents-aea/tree/main/plugins/aea-crypto-fetchai",
+    "https://github.com/fetchai/agents-aea/tree/main/plugins/aea-crypto-ethereum",
+    "https://github.com/fetchai/agents-aea/tree/main/plugins/aea-crypto-cosmos",
+}
+
```
Comment (marcofavorito) [inline scripts/check_doc_links.py:54]: temporary. It can be removed after the next release.

File: plugins/aea-crypto-fetchai/setup.py [hunk 2]
```diff
@@ -42,4 +42,23 @@
...
+        "Operating System :: Microsoft",
+        "Operating System :: Unix",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Topic :: Communications",
+        "Topic :: Internet",
+        "Topic :: Software Development",
+        "Framework :: AEA",
```
Comment (marcofavorito) [inline plugins/aea-crypto-fetchai/setup.py:62]: `Framework :: AEA` was kind of missing. It is also used by Pytest:
https://docs.pytest.org/en/latest/writing_plugins.html#making-your-plugin-installable-by-others

Comment (marcofavorito) [reply plugins/aea-crypto-fetchai/setup.py:62]: the others are copy-pasted from main `setup.py`, I have only removed some `Topic`s (Scientific/Engineering, System)

File: scripts/check_doc_links.py (hunk 1 above)
Comment (DavidMinarsch) [reply scripts/check_doc_links.py:54]: Please open issue so we don't forget

File: plugins/aea-crypto-cosmos/setup.py [hunk 3]
```diff
@@ -36,4 +36,22 @@
         "aea.ledger_apis": ["cosmos = aea_crypto_cosmos:CosmosApi"],
         "aea.faucet_apis": ["cosmos = aea_crypto_cosmos:CosmosFaucetApi"],
     },
+    classifiers=[
+        "Environment :: Console",
+        "Environment :: Web Environment",
+        "Development Status :: 2 - Pre-Alpha",
```
Comment (DavidMinarsch) [inline plugins/aea-crypto-cosmos/setup.py:42]: We should change this status when we do v1. Let's open an issue so we don't forget.

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**